package.json

```diff
@@ -56,13 +56,13 @@
     "@apollo/federation": "0.38.1",
     "@apollo/gateway": "0.50.2",
     "@aws-sdk/client-dynamodb": "3.415.0",
-    "@aws-sdk/client-dynamodb2": "npm:@aws-sdk/client-dynamodb@3.20.0",
+    "@aws-sdk/client-dynamodb-v3": "npm:@aws-sdk/client-dynamodb@3.20.0",
     "@aws-sdk/client-kinesis": "^3.379.1",
     "@aws-sdk/client-s3": "3.379.1",
-    "@aws-sdk/client-s3-2": "npm:@aws-sdk/client-s3@3.321.1",
+    "@aws-sdk/client-s3-v3": "npm:@aws-sdk/client-s3@3.321.1",
```

Comment:
_nitpick:_ The other version is also v3 (3.x.x) though. Just different minor versions of 3.x.

Reply:
See https://github.com/instana/nodejs/commit/3936a9da70752102e90cf3721e63bd83dce92151
This was the bigger smithy client refactoring from aws sdk.

Reply:
Sorry, I didn't mean to imply we should only test one particular v3 version. I was talking about the label we use for that additional version in the package.json file. Since both are v3, labeling one of the with the suffix "-v3" is a bit misleading/confusing. But whatever... it is only a label that we use internally.

Reply:
Ah okay. Any better idea? This label follows our pattern of `-vN`.